tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DbScopeFactory.cs

```diff
@@ -55,13 +58,19 @@ private static Scope CreateDbCommandScope(Tracer tracer, IDbCommand command, Int
                 tags.SetAnalyticsSampleRate(integrationId, tracer.Settings, enabledWithGlobalSetting: false);
 
                 scope = tracer.StartActiveInternal(operationName, tags: tags, serviceName: serviceName);
-                scope.Span.ResourceName = command.CommandText;
+                scope.Span.ResourceName = commandText;
                 scope.Span.Type = SpanTypes.Sql;
                 tracer.TracerManager.Telemetry.IntegrationGeneratedSpan(integrationId);
 
                 if (Iast.Iast.Instance.Settings.Enabled)
                 {
-                    IastModule.OnSqlQuery(command.CommandText, integrationId);
+                    IastModule.OnSqlQuery(commandText, integrationId);
+                }
+
+                if (tracer.Settings.DbmPropagationMode != DbmPropagationLevel.Disabled && (integrationId == IntegrationId.MySql || integrationId == IntegrationId.Npgsql))
+                {
+                    command.CommandText = $"{DatabaseMonitoringPropagator.PropagateSpanData(tracer.Settings.DbmPropagationMode, tracer.DefaultServiceName, scope.Span)} {commandText}";
+                    scope.Span.SetTag(Tags.DbmDataPropagated, "true");
```

Comment:
Now that you've added a new propergy `SqlTags.DbmDataPropagated` you can change this line to the following:
```suggestion
                    tags.DbmDataPropagated = "true";
```